app/src/modules/settings/routes/data-model/field-detail/shared/related-field-select.vue

```diff
@@ -6,7 +6,7 @@
 		:disabled="disabled"
 		:placeholder="placeholder"
 		:class="{ matches: fieldExists }"
-		@update:model-value="$emit('update:modelValue', $event.length ? $event : null)"
+		@update:model-value="$emit('update:modelValue', $event)"
```

Comment:
The previous condition is no longer necessary since we are passing `nullable`:
https://github.com/directus/directus/blob/f9c2b455b152aebe3e47598199211877898641b1/app/src/modules/settings/routes/data-model/field-detail/shared/related-field-select.vue#L5-L9